src/repl.jl

```diff
@@ -283,10 +283,13 @@ end
 NEW_KEYBINDINGS = create_keybindings()
 
 function insert_keybindings(repl = Base.active_repl)
-    mirepl = isdefined(repl,:mi) ? repl.mi : repl
-    main_mode = mirepl.interface.modes[1]
-    p = mirepl.interface.modes[5]
+    mirepl = isdefined(repl,:mistate) ? repl.mistate : repl
+    interface_modes = mirepl.interface.modes
+    main_mode = interface_modes[1]
+    php_idx = findfirst(isa.(interface_modes, LineEdit.PrefixHistoryPrompt))
```

Comment:
```suggestion
    php_idx = findfirst(isa(LineEdit.PrefixHistoryPrompt), interface_modes)
```
Avoid allocating

Reply:
Currently, `isa` requires two arguments explicitly. Here is a workaround:

```julia
findfirst(Base.Fix2(isa, LineEdit.PrefixHistoryPrompt), interface_modes)
```

However, I'm uncertain about using `Base.Fix2` since it hasn't been exported.

Reply:
Good point. It's fine to use Fix2, it's public, just not exported https://docs.julialang.org/en/v1/base/base/#Base.Fix2